modules/cli/cmd/devrunner/utilities/utilities.go

```diff
@@ -79,3 +80,21 @@ func ShellExpand(value string) string {
 	}
 	return expanded
 }
+
+// IsValidPort checks if the given port is a valid port number.
+//
+// Pancis
+func IsValidPortOrPanic(port string) {
+	// Convert string to integer
+	portNum, err := strconv.Atoi(port)
+	if err != nil {
+		log.Errorf("Error converting port to integer: %v", err)
```

Comment:
You can combine this line and the next line I think with something like `log.Fatal` or log.Panic`